regression/rra/test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Why is this script needed?  It feels like it is replicating some of the functionality of the test infrastructure.